flake.nix

```diff
@@ -1,77 +1,91 @@
 {
   inputs = {
```

Comment:
The diff is not very useful here. The file is small and most of it changed, so I suggest reviewing it as a whole.